### PR TITLE
standardize on df variable name

### PR DESCRIPTION
**Pull request by nicolaskruchten**

### CODE REVIEW

File: doc/python/px-arguments.md [hunk 1]
```diff
@@ -189,9 +190,9 @@ import plotly.express as px
 import numpy as np
 import pandas as pd
 
-gapminder = px.data.gapminder()
-gdp = np.log(gapminder['pop'] * gapminder['gdpPercap'])  # NumPy array
-fig = px.bar(gapminder, x='year', y=gdp, color='continent', labels={'y':'log gdp'},
+df = px.data.gapminder()
+gdp = np.log(df['pop'] * gapminder['gdpPercap'])  # NumPy array
```
Comment (emmanuelle) [inline doc/python/px-arguments.md]: gapminder -> df

Comment (emmanuelle) [reply doc/python/px-arguments.md]: (thanks to the CI !)

Comment (nicolaskruchten) [reply doc/python/px-arguments.md]: yep, just saw that one in the tests. I think it's the only one I missed though!

**Comment by nicolaskruchten** [comment]
FYI my search'n'replace approach only targets the datasets in `px.data` and not others we may have named while loading from files etc... should be a pretty good start though!

Do you recall any other names we may have used repeatedly?

**Comment by emmanuelle** [comment]
I did a `grep "pd.read_csv('https" *` and only found a couple of places where it's not `df` or `df_something`, like `milk_data` in `peak_finding.md`

**Pull request merged**